backend/aopp_test.go

```diff
@@ -183,4 +202,115 @@ func TestAOPPSuccess(t *testing.T) {
 			)
 		})
 	}
+
+	t.Run("error/wrong_version", func(t *testing.T) {
```

Comment:
All these belong to a new test, not `TestAOPPSuccess`:

```go
func TestAOPPErrors(t *testing.T) {
  t.Run("wrong version", func(...) {
  })
}
```

so, no need for `error/` prefix in the t.Run names.